src/display/api.js

```diff
@@ -2329,6 +2344,7 @@ class WorkerTransport {
     this._fullReader = null;
     this._lastProgress = null;
     this.downloadInfoCapability = Promise.withResolvers();
+    this.enableHWA = enableHWA;
```

Comment:
It's not entirely clear to me why this member variable, and the constructor change, is necessary given that I can't find a place where this is read. What am I missing here?

Reply:
So it's needed atm in `InternalRenderTask` in order to make the necessary `getContext` call and since that's called by `PDFPageProxy#render`, I'm storing it on `WorkerTransport` in order to pass it on here: https://github.com/mozilla/pdf.js/pull/20016/files#diff-082d6b37ad01db7ac97cc07c6ddb0dc52040484c5ef91b110b072f50144d9f39R1520.

In my follow-on PR (already up but in draft state at #20053), you would notice that the `getContext` call moves to a worker thread where I set `enableHWA` at the beginning once so I'll remove it from `WorkerTransport` as part of that patch.